### DISCUSSION TITLE
Radeon RX 6700 XT GPU Memory Junction Temperature sensor

### DISCUSSION THREAD

**Discussion by benoe77** [LibreHardwareMonitor/LibreHardwareMonitor#419] [Ideas]
Hi, could you please add the ram temperature sensor data of the new radeon RX 6700 XT? On 5700 it was correctly displayed, but after I changed the card, this information is missing.

Please let me know if I can help somehow adding it

Btw I'm using Remi Mercier's Fan Control, but as I understand, sensor data comes from LibreHardwareMonitor.

**Comment by whoozy82** [comment]
Same with the RX 6600 XT. It does not show Memory Temps nor VDDC Temps.
Would be great if it would be added.

Thanks
Kay
